# BETSE-repo
 betse repository final
 
 The ability to successfully run pipeline requires Matlab, Anaconda, WEKA, and CellProfiler; The repository is comprised of two main sections: 

•	Quantitative Image Analysis and Pattern Recognition

•	Parameter Optimization

The data and code pertaining to the ‘Quantitative Image Analysis and Pattern Recognition’ section is stored in the ‘../QuantitativeImageAnalysis/’ sub-directory and the data and code pertaining to ‘Parameter Optimization’ is stored in the ‘../temp_synthesis/’ sub-directory. Instructions to run the code are located in the ‘BETSE-repo/BETSE GitHub Instructions.docx’ file. The raw scRNA-sequencing data is located in the '../single cell RNA Seq data/' sub-directory, with the processed samples being located within the '../single cell RNA Seq data/Andre scRNAseq/' sub-directory in excel spreadsheets (.xlsx) in accordance with phenotype (PP-pluripotent, CM-cardiac mesoderm, CP-cardiac progenitor) and media treatment conditions (U vs. T, or untreated vs treated, respectively). The majority of analysis in this study was conducted on the untreated pluripotent sample.

The image sets associated with the computational simulations are stored in culture condition-specific folders in the '../QuantitativeImageAnalysis/BETSE Sim output image directories' sub-directory. The image sets associated with the experimental DiBAC fluorescent images and their transformed counterparts are stored in the '../QuantitativeImageAnalysis/2021 in vitro outcomes' sub-directory in condition-specific folders.

